### PR TITLE
fix for https://github.com/thecodingmachine/workadventure/issues/1256

### PR DESCRIPTION
**Pull request by jonnytest1**
- layers can be invisible (and will not be added)
- layers can have alpha(opcacity)

### CODE REVIEW

**Comment by jonnytest1** [comment]
@moufmouf

**Comment by Kharhamel** [comment]
Hello, thanks for the PR. Could you fix the conflicts and remove the changes to the linters from this PR? I will merge this once this is done.

File: front/.eslintrc.json [hunk 1]
```diff
@@ -24,9 +24,12 @@
         "@typescript-eslint"
     ],
     "rules": {
+        "quotes": [
```
Comment (Kharhamel) [inline front/.eslintrc.json:27]: please don't change this file in a "fix" PR

File: front/.prettierrc.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
-  "printWidth": 120,
```
Comment (Kharhamel) [inline front/.prettierrc.json:2]: please don't change this file in a "fix" PR

**Review by Kharhamel — CHANGES_REQUESTED** [review]

**Comment by Chasethechicken** [comment]
Hey @jonnytest1, sorry for pinging you, but PR this would be really useful.
Are you still planning on fixing it, or would you prefer if somebody else tackled #1256?

**Comment by jonnytest1** [comment]
> Hey @jonnytest1, sorry for pinging you, but PR this would be really useful.
> Are you still planning on fixing it, or would you prefer if somebody else tackled #1256?

Feel free to do it yourself :) I'm currently on vacation anyways ^^ 
PS u can probably just remove the questionable linter changes and re-request it 
😅

**Pull request closed without merging**